### PR TITLE
Feature/add titan

### PR DESCRIPTION
**Pull request by jdmedlock**
Add documentation on Titan `/user` command and ticketing system.

### CODE REVIEW

File: docs/resources/techresources/assets/Newly_opened_ticket.png
Comment (ellrub) [inline docs/resources/techresources/assets/Newly_opened_ticket.png:1]: FYI - This is mee6, not Titan :P

File: docs/resources/techresources/assets/Open_support_ticket.png
Comment (ellrub) [inline docs/resources/techresources/assets/Open_support_ticket.png:1]: Again, this is mee6, not Titan

File: docs/resources/techresources/titan.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+You can access information about your status, including your Solo Project and
+most recent Voyage signup using the `/user` command. You may issue this command
+in any of our Discord channels, but its output will only be visible to you.
+
+![User Command Output](./assets/User_command_output.png)
+
+## Using the Chingu Ticketing System
+
+To open a ticket, open the `#support-ticket-channel` and click on the 
+`Create Ticket` button. Titan will then open a thread inside the
```
[the remark this answers is not in the document]
Comment (ellrub) [reply docs/resources/techresources/titan.md:23]: a little confusing now that both bots have the ticket system open, but I think it we'll have enough information about the stability before voyage 49, and we can then make a decision whether or not Titan is ready to handle it on it own

[the remark this answers is not in the document]
Comment (jdmedlock) [reply docs/resources/techresources/titan.md]: @ellrub I've updated the Titan ticketing screenshots and description to correct these issues. Thanks for your feedback on this and let me know if you see any new issues. 

This PR also contains new PO & SM role definitions, which I'll have Bahareh, Christopher, & Zahra review once the Titan review is complete.

**Review by ellrub — APPROVED** [review]

**Pull request merged**